api/src/test/java/edu/cornell/mannlib/vitro/webapp/dynapi/ResourceAPIPoolTest.java

```diff
@@ -0,0 +1,552 @@
+package edu.cornell.mannlib.vitro.webapp.dynapi;
```

Comment:
When not using git mv, the commit history is degraded as not able to easily find this file history.